### DISCUSSION TITLE
The client ip obtained by the rate-limiting plugin is 127.0.0.1

### DISCUSSION THREAD

**Discussion by xcrzh** [Kong/kong#8507] [Help]
Both in the log and in the database are 127.0.0.1

![image](https://user-images.githubusercontent.com/35281450/156892490-0f62a297-105d-4ac1-bcce-789aa784ebf6.png)
![image](https://user-images.githubusercontent.com/35281450/156892529-eb813cda-8905-4168-be2e-333c63fa0801.png)

**Comment by xcrzh** [comment]
### Summary
I deployed kong on kubernetes, used NodePort to expose ports 80 and 443, and then configured the domain name on kong, which was able to access normally, but could not get the real IP of the client.

In front of kong is Aliyun SLB for the reverse.

### Steps To Reproduce
1. I configured the following environment variables:
```
KONG_TRUSTED_IPS: 0.0.0.0/0,::0
KONG_REAL_IP_RECURSIVE: on
KONG_REAL_IP_HEADER: X-Forwarded-For
```

2. Finally, I accessed the configured domain name, but the kong log did not get the IP of my computer.

### Additional Details & Logs
kong image version: kong/kong-gateway:2.8.0.0-alpine
k8s version: 1.20.4

**Comment by StarlightIbuki** [comment]
Can you dump and post the request? especially: 
1. IP address of the last hop;
2. X-Forwarded-For header

Thank you.

**Reply by StarlightIbuki** [reply]
Also, there seems to be a mistake in your configuration: `::0`, maybe you mean `::/0`?
